torch/testing/_internal/common_utils.py

```diff
@@ -1770,10 +1773,17 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, hig
         result = torch.repeat_interleave(result, 2, dim=-1)
         result = result[..., ::2]
 
+    if not include_zero and dtype not in complex_types():
```

Comment:
Should we extend this approach to complex numbers, too?

Reply:
Done! Thanks for pointing out.